dashboard/app/models/pd/workshop.rb

```diff
@@ -519,6 +519,10 @@ def teachercon?
     subject == SUBJECT_TEACHER_CON
   end
 
+  def summer?
```

Comment:
heh - I just wrote literally the same helper method but called it `summer_workshop?`